tests/TestTag.shadow-dom.test.js

```diff
@@ -9,11 +9,15 @@ import { syncRaf } from './test-utils.js';
 // TODO: Needs unit tests to validate attributes/props being handled correctly (at minimum pass through)
 
 let el = null;
+let cssFile = 'test.css';
+let cssFiles = ['test0.css', 'test1.css', 'test2.css'];
```

Comment:
IMHO since these _probably_ aren't being reused, it's best to just define inline in the `svelteRetag()` call, just for clarity, since they aren't reused/redefined.